docs/docusaurus/docs/cloud/expectations/manage_expectations.md

```diff
@@ -52,17 +62,17 @@ The following table lists the available GX Cloud Expectations.
 
 5. Select an Expectation type. See [Available Expectation types](#available-expectation-types).
 
-6. If you are adding your first expectation on this data asset, you may be able to select a time-based Batch interval for that asset. 
+6. If you are adding your first expectation on this data asset, you may be able to select a time-based Batch interval for that asset.
```

Comment:
I trimmed trailing whitespace and it did it to the whole file. That's the only change for the for all of the following lines.